junit-jupiter-params/src/main/java/org/junit/jupiter/params/converter/ArgumentConverter.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.jupiter.params.converter;
+
+import org.junit.jupiter.api.extension.ParameterContext;
+
+public interface ArgumentConverter {
```

Comment:
I was thinking as an _idealist_ for what I think a `Converter` API should look like, but now that I've looked at the code in the IDE... I agree with you: it's best to leave off the generics in this particular case.